wagtailsharing/tests/test_checks.py

```diff
@@ -2,19 +2,26 @@
 from django.core.checks import Error
 from django.test import SimpleTestCase, override_settings
 
-from wagtailsharing.checks import modeladmin_installed_check
+from wagtail import VERSION as WAGTAIL_VERSION
 
 
-class TestModelAdminInstalledCheck(SimpleTestCase):
-    @override_settings(
-        INSTALLED_APPS=["wagtail.contrib.modeladmin", "wagtailsharing"]
-    )
-    def test_check_passes_if_modeladmin_installed(self):
-        self.assertFalse(modeladmin_installed_check(apps.get_app_configs()))
+if WAGTAIL_VERSION <= (5, 0):
+    from wagtailsharing.checks import modeladmin_installed_check
 
-    @override_settings(INSTALLED_APPS=["wagtailsharing"])
-    def test_check_fails_if_modeladmin_not_installed(self):
-        errors = modeladmin_installed_check(apps.get_app_configs())
-        self.assertEqual(len(errors), 1)
-        self.assertIsInstance(errors[0], Error)
-        self.assertEqual(errors[0].id, "wagtailsharing.E001")
+if WAGTAIL_VERSION <= (5, 0):
+
+    class TestModelAdminInstalledCheck(SimpleTestCase):
```

Comment:
These conditionals can be merged:

```
if WAGTAIL_VERSION <= (5, 0):
    from wagtailsharing.checks import modeladmin_installed_check

    class TestModelAdminInstalledCheck(SimpleTestCase):
```

Optionally, other imports can also be under this conditionals since only code for Wagtail <= 5.0 uses those imports.

Reply:
Thats a good idea thanks.